tests/admin_inlines/tests.py

```diff
@@ -854,6 +855,49 @@ def test_delete_inlines(self):
         self.assertEqual(len(self.selenium.find_elements_by_css_selector(
             'form#profilecollection_form tr.dynamic-profile_set#profile_set-2')), 1)
 
+    def test_gone_deletes(self):
+        """
+        Ensure that the delete buttons are also present in the form after
+        a form validation error. Bug #15910.
```

Comment:
We don't really need to include the reference to the ticket except in obscure situations where the behavior could benefit from additional explanation that doesn't fit in a docstring. I don't think that's the case here.